tests/installation/bootloader_s390.pm

```diff
@@ -176,18 +178,21 @@ EO_frickin_boot_parms
     ## an empty line and a single "manual=1" at the bottom
     ## of the ftpboot parmfile.  This may fail in obscure
     ## ways when that changes.
-    $s3270->sequence_3270(qw(String(BOTTOM) ENTER String(DELETE) ENTER));
-    $s3270->sequence_3270(qw(String(BOTTOM) ENTER String(DELETE) ENTER));
+    unless (get_var('AGAMA')) {
```

Comment:
why is this needed? seems that this code is there for some sporadic reasons, it works in O3 that I know.

Reply:
In Agama the `manual=1` is not present so to avoid last line removal we should not delete the last line of the parmfile.

Reply:
then perhaps a comment would help.

Reply:
21160037a